src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepositorySpec.scala

```diff
@@ -103,4 +101,73 @@ class ConsignmentRepositorySpec extends AnyFlatSpec with TestDatabase with Scala
     sequenceId should be(expectedSeq)
   }
 
+  "getConsignment" should "return the consignment given the consignment id" in {
+    val db = DbConnection.db
+    val consignmentRepository = new ConsignmentRepository(db, new CurrentTimeSource)
+
+    TestUtils.createConsignment(consignmentIdOne, userId)
+
+    val response = consignmentRepository.getConsignment(consignmentIdOne).futureValue
+
+    response.size should be (1)
+    response.headOption.get.consignmentid should equal(consignmentIdOne)
+  }
+
+  "getConsignments" should "return all consignments including cursor value up to the limit value" in {
+    val db = DbConnection.db
+    val consignmentRepository = new ConsignmentRepository(db, new CurrentTimeSource)
+    createConsignments()
+
+    val response = consignmentRepository.getConsignments(2, "TDR-2021-B").futureValue
+
+    response.size should be (2)
+    val consignmentIds: List[UUID] = response.map(cr => cr.consignmentid).toList
+    consignmentIds.contains(consignmentIdTwo) should be (true)
+    consignmentIds.contains(consignmentIdThree) should be (true)
+  }
+
+  "getConsignments" should "return all consignments up to limit where empty cursor provided" in {
+    val db = DbConnection.db
+    val consignmentRepository = new ConsignmentRepository(db, new CurrentTimeSource)
+    createConsignments()
+
+    val response = consignmentRepository.getConsignments(2, "").futureValue
```

Comment:
As above, using an empty string is a bit hard to follow because the meaning is not obvious from the type system.

If the empty string _is_ the Relay convention, could we convert it into an `Option` in the service layer so the repository can deal with a cursor that's got a type of `Option[String]` that makes it more obvious that it is sometimes empty?